css-color-markers/styles.css

```diff
@@ -0,0 +1,60 @@
+h1 {
+  text-align: center;
+}
+
+.container {
+  background-color: rgb(255, 255, 255); /* Just to teach additive color combinations */
+  padding: 10px 0;
+}
+
+.marker {
+  width: 200px;
+  height: 25px;
+  margin: 10px auto;
+}
+
+.tip {
+  width: 35px;
+  height: 25px;
+  display: inline-block;
+}
+
+.sleeve {
+  width: 110px;
+  height: 25px;
+  display: inline-block;
+  vertical-align: top;
+  text-align: left;
+  padding-left: 10px;
+  font-size: 20px;
+  border-left: 10px double rgba(0, 0, 0, 0.75);
+}
+
+.marker > .sleeve {
```

Comment:
A quick note that this will be the first time the `>` selector is introduced.

Reply:
Awesome, thanks for pointing this out.

Ahmad mentioned this to me in the past. I'm just getting around to that point in the steps, so I'll make sure to spend extra time on the `>` selector.

I'll also update the spreadsheet to reflect that now.